src/Impl/RestPublisher.php

```diff
@@ -15,27 +14,32 @@
 
 class RestPublisher extends Publisher
 {
-    private const INGESTION_ENDPOINT_RELATIVE_PATH = "channels/[CHANNEL]/messages";
-    private const SCHEMA_ENDPOINT_RELATIVE_PATH = "channels/[CHANNEL]/schema";
+    private const HEALTH_CHECK_ENDPOINT_PATH_TEMPLATE = '%s/q/health';
+    private const INGESTION_ENDPOINT_PATH_TEMPLATE = '%s/channels/%s/messages';
+    private const SCHEMA_ENDPOINT_PATH_TEMPLATE = '%s/channels/%s/schema';
```

Comment:
Refactor handling the two existing paths, because the new one doesn't contain the `/ingestion/v1` token